provisioning/roles/kuma/files/apache2/all-servers.conf

```diff
@@ -3,15 +3,6 @@
 # Our servers symlink to this file. Do not move it without first coordinating
 # with WebOps.
 
-<Location /static/fonts>
-    # Only enable CORS for fonts when the request is coming from an approved domain
-    Header unset Access-Control-Allow-Origin
-    SetEnvIf Origin "https?://(.*\.(mozilla|allizom)\.(com|org|net))" CORS=$0
-    SetEnvIf Origin "https?://(mdn\.mozillademos\.org)" CORS=$0
-    SetEnvIf Origin "https?://(kuma\.readthedocs\.org)" CORS=$0
-    Header set Access-Control-Allow-Origin %{CORS}e env=CORS
```

Comment:
Great. Thanks for pointing me to that documentation.

My understanding is that these lines were added to prevent others from hotlinking licensed fonts. There doesn't seem to be any documentation supporting that, though, so I'll assume it's safe to disable that protection unless someone knows otherwise.